subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.c

```diff
@@ -334,6 +334,547 @@ psa_status_t cracen_ecc_reduce_p256(const uint8_t *input, size_t input_size, uin
 	return silex_statuscodes_to_psa(sx_status);
 }
 
+psa_status_t cracen_ecc_is_quadratic_residue(const sx_const_op *curve_prime,
```

Comment:
Is this used for anything except WPA3? It might anyways be worth moving into either ec_helpers, ecc or a new file since common.c is already quite chunky.